### DISCUSSION TITLE
New York, NY, USA DataFusion Meetup: Tuesday Sep 17th 2024, 5:30PM - 8:30PM Manhattan

### DISCUSSION THREAD

**Discussion by alamb** [apache/datafusion#11213] [General]
Follow on to https://github.com/apache/datafusion/discussions/10343

We are organizing a NYC meetup. Registration Link: https://lu.ma/2iwba0xm

Tuesday Sep 17th 2024, 5:30PM - 8:30PM 

Address:  New York Times Building, 620 8th Ave, NY 10018

Will need a full guest list 3 days before scheduled event with Full Name, Email, Company.

**Comment by alamb** [comment]
Is anyone in the NYC area is willing to speak for 10-15 minutes at this meetup? I would suggest topics like 
1. "What you are doing with DataFusion"
2. "What are some thing that are great and no so great"
3. "What DataFusion feature are you working on lately" (or a pitch to get others excited about a feature you would like)

Maybe @matthewmturner @edmondop @suremarc @philippemnoel might be interested or know of people who might be ?

**Reply by gforsyth** [reply]
Hey @alamb -- I'm in the area and if y'all would be interested in an ibis+datafusion talk I'd be happy to present.

**Reply by alamb** [reply]
That would be awesome -- thank you -- please email me at alamb@influxdata.com and we'll arrange the details

**Reply by edmondop** [reply]
I won't be able to speak, but I'd love to help with the organization. If people are interested I can find a venue too

**Reply by alamb** [reply]
Thanks @edmondop  -- I'll add you as an organizer to the event

**Comment by matthewmturner** [comment]
It looks like were going to plan a team meetup in New York around the DataFusion meetup and we will very likely have a talk on what we are doing with DataFusion.  Will keep you posted as our plans develop / finalize.

**Comment by kylebarron** [comment]
I'll be out of town but will definitely join the next meetup you have in NYC!

**Comment by alamb** [comment]
Here is the registration Link: https://lu.ma/2iwba0xm (also added to this the description)

**Reply by alamb** [reply]
Slides from the meetup:


[Composing Data Systems At Datadog.pdf](https://github.com/user-attachments/files/17080025/Composing.Data.Systems.At.Datadog.pdf), - Wendell Smith and Alex Bianchi, DataDog.

[​"Ibis + DataFusion playing (very) nicely together."](https://ibis-project.org/presentations/datafusion-meetup-nyc-2024/talk)- Gil Forsyth, Voltron Data, Senior Staff Software Engineer

[​"Using DataFusion to build InfluxDB 3.0"](https://docs.google.com/presentation/d/1dOLPAFPEMLhLv4NN6O9QSDIyyeiIySqAjky5cVgdWAE/edit#slide=id.g26bebde4fcc_3_7): Andrew Lamb, InfluxData, Staff Engineer

[​"Materialized Views and Query Rewriting in DataFusion"](https://drive.google.com/open?id=1mHDw1uZcOwlpUO3mA8aqSyk7IqeovpSuXG27clowXWE): Matthew Cramerus, [Polygon.io](http://polygon.io/), Software Engineer


[​"Embedding DataFusion in Postgres, Strengths and Limitations"](https://docs.google.com/presentation/d/15yZBgAKSUB8nQGTOg9hNzuSAN7D9SPPAs_mpyxqoZyk/edit?usp=sharing): Philippe Noël, ParadeDB, CEO


[​"dft: A terminal application for DataFusion":](https://docs.google.com/presentation/d/1u42k8ZBHObLx1Ph5CvQOkeRJ72ys81Jo7BkCxjCU5Ik/edit#slide=id.p) Matthew Turner

**Reply by JanKaul** [reply]
Are there any recordings of the videos? I would love to see what the community is up to

**Reply by alamb** [reply]
We don't have recordings unfortunately, just the slides
